# isomiR2Function
isomiR2Function allows for the following enhanced functionalities: 1. It allows for customizable length range, seed corrupt tolerance and minimum sequencing depth for isomiR identification; 2. Identification of isomiR on pre-miRNA not overlapping with canonical miRNAs; 3. Information on isomiR biogenesis using Ks plots; 4. Support for differential expression analysis of the identified isomiRs using DESeq (Anders and Huber 2010) and EBSeq (Leng et al., 2015); 5. Detailed classification of multiple internal substitution. isomiR2Function classifies isomiRs with multiple internal substitution in more detail, i.e. MS for having internal random SNPs, CV for having both internal SNPs and terminal substitutions and TS for having internal tandem SNPs; 6. Support for transcript based target identification using TargetFinder (Fahlgren et al., 2007) or degradome based PARE target identification using Cleaveland (Zhang et al., 2014); 7. Functional enrichment of the isomiR targets for biological implications.
